build.py

```diff
@@ -5,7 +5,8 @@
 from bincrafters import build_template_default
 
 if __name__ == "__main__":
-
-    builder = build_template_default.get_builder()
-
+    builder = build_template_default.get_builder(
+        pure_c=True,
+        build_policy="missing"
```

Comment:
Please don't do this. If prebuilds are missing let us know so we can fix it

Reply:
I've reverted it. There seems to be no ICU build [for linux gcc 9](https://travis-ci.com/bincrafters/conan-mongo-c-driver/jobs/288331658).